public/app/plugins/panel/table/TablePanel.tsx

```diff
@@ -41,14 +42,16 @@ export function TablePanel(props: Props) {
   const tableElement = (
     <Table
       height={tableHeight}
-      width={width}
+      // This calculation is to accommodate the optionally rendered Row Numbers Column
+      width={options.showRowNums ? width : width + OPTIONAL_ROW_NUMBER_COLUMN_WIDTH}
```

Comment:
Is there a specific reason we need to change width here? I _think_ we can leave it as it was. The variable holds the width of the panel and is unrelated to the width of the table itself, which will fit into the panel and the overflow will be scrollable.

Reply:
Yep, without it, the table won't flex to the proper width when the row number column is not being rendered.

Reply:
@baldm0mma I really don't understand this logic :( 

The total width of the Table component should not change depending on if showRowNums is true or not 

Adding more width here is making it go beyond the bounds of it's container

Reply:
I believe I needed to do this because when we use the setHiddenColumns() hook from React Table - which I use to show/hide the table number row - the table calculations don't automatically account for the new table width and flex accordingly. Which is why when we hide the row, we see this empty space:
<img width="1066" alt="Screenshot 2023-03-28 at 8 50 46 AM (2)" src="https://user-images.githubusercontent.com/46619047/228279585-5dafdbc5-bed7-4254-abc0-8c4ab9380776.png">
This calculation was meant to deal with that.

Reply:
So when the number row is hidden, we need to add back the width of that hidden row.